pkg/api/v1beta3/conversion.go

```diff
@@ -3227,3 +2746,474 @@ func init() {
 		panic(err)
 	}
 }
+
```

Comment:
This was due to the change of removing a field from the internal api while retaining it in the v1beta* versions on the Container struct.  The conversion generator would not generate anything that had a field mismatch along with anything that contained the Container struct.  So yes, most of these are straight copies (retaining the naming convention as well).  The only actual meat in this file is the change to the container conversions.  The rest was a byproduct of the conversion utility no longer generating functions.

Important to note though, is that the functions must be moved.  The unit test for the auto generator uses the comments in the file to determine what to check to ensure that auto generated functions are not manually changed.  That is also why there is now the second call to Register the moved functions rather than leaving them in the RegisterAutoGenerated call.